CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Changed
 * Bumped RNSM to 1.1.0: Switched x axis units to seconds instead of frames now that it's available. This means adjusting the sample count to a lower value as well to 30 seconds, since the x axis was moving too slowly. (#788)
 * Updated Boss Room to NGO 1.2.0 (#791).
+  * Removed a workaround in our tests waiting for two frames before shutting down a client that is attempting to connect to a server. (see https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/2261)
+
+### Cleanup
```

Comment:
This feels bigger than just a cleanup, this should be at the front of our "changed" or "added" section.